rpc_jobs/rpc_octavia.yml

```diff
@@ -7,7 +7,7 @@
     # currently we only do master. Once RPC-O switches
     # to Pike we will retire os-octavia and use the
     # upstream installer
-    branches:
+    branch:
```

Comment:
There is no need to specify branch for PR jobs as they get their checkout using details provided by github. ``branches`` is specified as a list of branches to allow builds to be triggered from, rather than as an axis as one would need in a periodic job.

Reply:
Removed.